libraries/Microsoft.Bot.Builder.AI.LUIS/LuisAdaptiveRecognizer.cs

```diff
@@ -40,6 +40,13 @@ public LuisAdaptiveRecognizer()
         [JsonProperty("applicationId")]
         public StringExpression ApplicationId { get; set; }
 
+        /// <summary>
+        /// Gets or sets LUIS version.
+        /// </summary>
+        /// <value>application version.</value>
+        [JsonProperty("version")]
+        public StringExpression Version { get; set; }
```

Comment:
Why is this needed as part of this PR?

Reply:
And this is a bad idea--version is already in prediction options.

---
In reply to: [480166975](https://github.com/microsoft/botbuilder-dotnet/pull/4537#discussion_r480166975) [](ancestors = 480166975)

Reply:
not supposed to be, got my PRs wires crossed.


---
In reply to: [481307166](https://github.com/microsoft/botbuilder-dotnet/pull/4537#discussion_r481307166) [](ancestors = 481307166,480166975)